charts/client/chart.scss

```diff
@@ -861,32 +862,41 @@ $zindex-IndicatorDropdown: 50;
             }
         }
 
-        $handle-width: 0.5em;
+        $handle-diameter: 16px;
         .slider {
-            height: 0.4em;
+            height: 4px;
             flex-grow: 1;
-            background: #eee;
+            background: #adb6c0;
             border-radius: 5px;
             position: relative;
             display: flex;
             align-items: center;
-            margin-right: 2 * $handle-width;
+            margin-left: $handle-diameter;
+            margin-right: $handle-diameter;
         }
 
         .handle {
-            height: 1em;
-            border-radius: 0.2em;
-            width: $handle-width;
+            height: $handle-diameter;
+            width: $handle-diameter;
+            border-radius: 100%;
             background: $controls-color;
             position: absolute;
-            cursor: col-resize;
+
```

Comment:
You can just add `margin-left: -$handle-diameter / 2;` here and won't need the whole computing of sizes in JS!

Reply:
Absolutely wonderful. Removed a lot of ugly code with that.